.github/workflows/wheels.yml

```diff
@@ -108,16 +114,22 @@ jobs:
           submodules: recursive
           path: "."
 
+      - name: Setup proper pytorch dependency in "requirements.txt"
+        run: >
+          sed -i '/pytorch/d' ./requirements.txt
+          echo "pytorch == ${{ matrix.config.torch_version_constraint }}" >> ./requirements.txt
+          cat ./requirements.txt
+
       - name: Install build dependencies
         run: >
           $PY -m pip install wheel setuptools torch==${{ matrix.config.torch_version }} -r requirements.txt --extra-index-url ${{ matrix.config.pip_index }} &&
```

Comment:
I wonder if `torch==${{ matrix.config.torch_version }}` is still necessary after doing the new update, would it make sense to just write the version with cuda in the requirements file and remove the torch version from here?

Reply:
Good point - will update